app/build.gradle.kts

```diff
@@ -1,196 +1,36 @@
 import com.google.gms.googleservices.GoogleServicesPlugin.GoogleServicesPluginConfig
 
 plugins {
-    alias(libs.plugins.android.application)
-    alias(libs.plugins.kotlin.android)
-    alias(libs.plugins.ksp)
-    alias(libs.plugins.kotlin.parcelize)
+    alias(libs.plugins.homeassistant.android.application)
     alias(libs.plugins.firebase.appdistribution)
     alias(libs.plugins.google.services)
-    alias(libs.plugins.hilt)
-    alias(libs.plugins.compose.compiler)
+    alias(libs.plugins.homeassistant.android.dependencies)
 }
 
 android {
-    namespace = "io.homeassistant.companion.android"
-
-    compileSdk = libs.versions.androidSdk.compile.get().toInt()
-
-    ndkVersion = "21.3.6528147"
-
     useLibrary("android.car")
 
     defaultConfig {
-        applicationId = "io.homeassistant.companion.android"
-        minSdk = libs.versions.androidSdk.min.get().toInt()
-        targetSdk = libs.versions.androidSdk.target.get().toInt()
-
-        versionName = project.version.toString()
-        versionCode = System.getenv("VERSION_CODE")?.toIntOrNull() ?: 1
-
         manifestPlaceholders["sentryRelease"] = "$applicationId@$versionName"
         manifestPlaceholders["sentryDsn"] = System.getenv("SENTRY_DSN") ?: ""
 
         bundle {
             language {
+                // We want to keep the translations in the final AAB for all the language
                 enableSplit = false
             }
         }
```

Comment:
I did not move this into the convention plugin based on the fact that it is not set for the `:wear` module. I'm not 100% but is it possible to application bundle for wear apps 🤔?

Reply:
What do you mean with "application bundle for wear apps"?

Everything submitted to Google Play is an aab. I don't believe you can change the language for apps on Wear OS independently of the OS which is why there is no reason to disable that split.

Reply:
I was not sure the app for Wear was an AAB or an APK. TBH last time I publish an app for Wear OS AAB was not a thing yet (or I was not using it).

Reply:
(If you take a look at the GitHub Actions run for the last beta you can see the files generated and pushed.)